src/diffusers/pipelines/pixart_alpha/pipeline_pixart_alpha.py

```diff
@@ -941,8 +941,12 @@ def __call__(
 
                 # compute previous image: x_t -> x_t-1
                 if num_inference_steps == 1:
-                    # For DMD one step sampling: https://arxiv.org/abs/2311.18828
-                    latents = self.scheduler.step(noise_pred, t, latents, **extra_step_kwargs).pred_original_sample
+                    try:
+                        # For LCM one step sampling
+                        latents = self.scheduler.step(noise_pred, t, latents, **extra_step_kwargs).denoised
+                    except:
```

Comment:
```suggestion
                    except Exception:
```
At minimum due to [`ruff`](https://github.com/huggingface/diffusers/actions/runs/13895274934/job/38874447777). What's the exception type raised here? Is there anything else we can check instead of relying on try/except?

Reply:
This one is due to the Output difference between
https://github.com/huggingface/diffusers/blob/100142586f82a9410f5bc393b9eb06c12d771006/src/diffusers/schedulers/scheduling_lcm.py#L48-L49
and 
https://github.com/huggingface/diffusers/blob/100142586f82a9410f5bc393b9eb06c12d771006/src/diffusers/schedulers/scheduling_ddim.py#L46-L47

What do you think is better to write here?

Reply:
gentle ping @hlky

Reply:
With `return_dict=False` a `tuple` is returned, both `denoised` and `pred_original_sample` are at index 1 so we can do
```python
latents = self.scheduler.step(noise_pred, t, latents, **extra_step_kwargs, return_dict=False)[1]
```

Reply:
That makes sense. updated.